src/Router.js

```diff
@@ -49,9 +47,17 @@ const Router = ({ createReducer, sceneStyle, scenes, navigator, getSceneStyle, c
     navigationStore.dispatch = dispatch;
     return <AppNavigator navigation={addNavigationHelpers({ dispatch, state })} />;
   }
-  return <App {...props} navigator={AppNavigator} />;
+
+  let WiredApp = App;
+
+  if (engine.appHoc) {
```

Comment:
I have it so you could skip the HOC if you wanted to. Not sure there will be a valid case for that yet.

Reply:
I think it's fine if it's possible and doesn't get in the way